### PR TITLE
chore: remove statements about exchange rate

### PR DESCRIPTION
**Pull request by karlwuest**

### CODE REVIEW

**Review by mlegner — APPROVED** [review]
LGTM. We can still decide if/when we want to merge this.

**Comment by karlwuest** [comment]
>We can still decide if/when we want to merge this.

I agree, I'll leave it as draft for now.

File: docs/blog/04_testnet_update.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Announcing Testnet
 
-Published on: 2024-10-17
+Published on: 2024-10-17 (updated on: 2024-12-04)
```
Comment (karlwuest) [inline docs/blog/04_testnet_update.md:3]: TODO: change date to actual date of update before merging.

**Comment by mlegner** [comment]
I'm closing this for now, we can revisit this if necessary.

**Pull request closed without merging**